lib/rucio/common/utils.py

```diff
@@ -783,7 +783,7 @@ def extract_scope(did, scopes=None, default_extract=_DEFAULT_EXTRACT):
     if not _loaded_policy_package_scope_algorithms:
         register_policy_package_algorithms('scope', _EXTRACT_SCOPE_ALGORITHMS)
         _loaded_policy_package_scope_algorithms = True
-    extract_scope_convention = config_get('common', 'extract_scope', False, None)
+    extract_scope_convention = config_get('common', 'extract_scope', False, None) or config_get('policy', 'extract_scope', False, None)
```

Comment:
@cserf in this case `extract_scope` in `common` gets a priority over the `extract_scope` in `policy` section. Should it be the other way around?

Reply:
This is for backward compatibility. I'll remove the common one later